app/domain/authentication/authn_gcp/validate_status.rb

```diff
@@ -0,0 +1,24 @@
+module Authentication
+  module AuthnGcp
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        discover_identity_provider:  Authentication::OAuth::DiscoverIdentityProvider.new
+      },
+      inputs:       %i()
+    ) do
+
+      def call
+        validate_provider_is_responsive
```

Comment:
this method does not validate it return true/false, maybe `provider_responsive?`?

Reply:
not quite. it calls a class that will return the discovered provider or will raise an error if it can't be discovered. so validate is good.